test/lib/ufe/testAttribute.py

```diff
@@ -192,7 +262,7 @@ def testAttributeGeneric(self):
         # Use our engine method to run the bulk of the test (all the stuff from
         # the Attribute base class). We use the xformOpOrder attribute which is
         # an unsupported USD type, so it will be a UFE Generic attribute.
-        ufeAttr, usdAttr = attrDict = self.runTestAttribute(
+        ufeAttr, usdAttr = self.runTestAttribute(
```

Comment:
I noticed this extraneous attrDict var that I think was a mistake during the initial commit.